src/main/java/io/vertx/mqtt/MqttServerOptions.java

```diff
@@ -100,6 +101,12 @@ public MqttServerOptions setHost(String host) {
   }
 
   @Override
+  public MqttServerOptions setClientAuth(ClientAuth clientAuth) {
+    super.setClientAuth(clientAuth);
+    return this;
+  }
```

Comment:
It looks like MqttClient just got SSL support too so will see how that could be utilised for testing. There seems to be some SSL test code in vert.x core too... Let's see what I can bake.